### PR TITLE
React-Redux

### PR DESCRIPTION
**Pull request by ReemMohamedAbdelfatah**
In this pull request the followings are done:

- Import Redux and connect the app to the Redux store.
- list of books displays books from Redux store.
- add functionality for "Add new" button click.
- add a functionality for "Remove" button click.
- add a functionality for "Check status" button click.

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -1,20 +1,56 @@
...
+        <select
+          className="form-select"
+          value={state.author}
+          onChange={handleTitleChange}
+        >
+          <option value="">Author</option>
+          <option value="option-1">Select option 1</option>
+          <option value="option-2">Select option 2</option>
+          <option value="option-3">Select option 3</option>
+        </select>
```
Comment (franclobo) [inline src/components/Form.js]: - [x] Kindly correct the input to add the author, it is supposed that the user will write the name of the author not select the author. May you want to select the category? Please 🙏🏽 correct this section of the form for a better user experience.

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@
 
 > Describe your project in 1 or 2 sentences.
 
-**[Book store]** "Book store" is an MVP version of it that allows you to:
+**[Book store using react-redux]** "Book store" is an MVP version of it that allows you to:
```
Comment (franclobo) [inline README.md:35]: - [x] Please 🙏🏽 correct the title of the project, it is Book Store, not Math magicians for a professional README file.
- [x] This is not the To-Do List app, please actualize the links on Your README file to have professional documentation. 👇🏽 

image


- [x] We are not using webpack for this project, please make sure to actualize each section on your README for a professional README file.

image

- [x] Kindly add the LICENSE file to your repo and make sure that the MIT link works as expected.

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
   <img src="mathmagician.jpeg" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Book Store</b></h3>
+  <h3><b>Book Store using React</b></h3>
```
Comment (franclobo) [inline README.md:5]: - [x] Please 🙏🏽 remove the Microsoft logo or replace it with your own logo for a professional README file.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @ReemMohamedAbdelfatah, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 1 above)
Comment (ReemMohamedAbdelfatah) [reply src/components/Form.js]: Done 👌

File: README.md (hunk 3 above)
Comment (ReemMohamedAbdelfatah) [reply README.md:5]: Done👌

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @ReemMohamedAbdelfatah👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ReemMohamedAbdelfatah** [comment]
Thank you!

**Pull request merged**